src/scripts/quests/QuestLineHelper.ts

```diff
@@ -1968,11 +1968,181 @@ class QuestLineHelper {
         App.game.quests.questLines().push(hoopaDayPikabluQuestLine);
     }
 
+    public static createMeltanQuestLine() {
```

Comment:
The questlines needs to be moved up to the alola questlines

Reply:
Merge conflicts resolved, eslint passing.

Reply:
Questlines moved into Alola section. Not sure what you mean by the indexes in the update functions. And idk if we have images for any of the temp battles yet (at the very least the two grunts, and maybe Faba? We should have the Leaders.)

Reply:
(ah, I see, completely missed those Update lines... not that I would've figured out what needed fixing)